.gitignore

```diff
@@ -5,16 +5,13 @@ schemadev/
 .coveralls.yml
 *.txt
 *.out
-*.yaml
 *test.sh
 utils/
 in/
 out/
 dev/
 t/
 
-test/_*/
```

Comment:
Can happily live with the first change, but this exception is specifically called out in the documentation so you can have 'private' tests. I have some from (cough) a well known office-space-sharing company, that I don't want to accidentally commit to the repo.